server/events/vcs/github_client.go

```diff
@@ -198,7 +198,7 @@ func (g *GithubClient) HidePrevPlanComments(repo models.Repo, pullNum int) error
 			continue
 		}
 		firstLine := strings.ToLower(body[0])
-		if !strings.Contains(firstLine, models.PlanCommand.String()) {
+		if !(strings.Contains(firstLine, models.PlanCommand.String()) || strings.Contains(firstLine, strings.ToLower("Continued from previous comment."))) {
```

Comment:
I'm don't usually program in Golang but I generally prefer not to use hardcoded values. However it seems that the text is hardcoded as well when generating the comments. Should I extract it in a constant or something?

Reply:
constant would be great.

Reply:
I took a crack at it and was able to simplify a bit of code, but again, I'm no Golang expert, so any feedback is welcome!